bindings/wasm/test/examples/tetrahedron-puzzle.mjs

```diff
@@ -70,3 +70,6 @@ moving.rotation = (t) => [0, 0, 270 * 2 * (1 - t)];
 
 globalDefaults.animationLength = 10;  // seconds
 globalDefaults.animationMode = 'ping-pong';
+
+export const nodes = [layFlat, fixed, layFlat2, moving];
```

Comment:
Not a critique, but for my own understanding - if this were just `const nodes = [fixed, moving]`, would it still work? Since `fixed` references `layFlat` as its parent? Or could you do `const nodes = [layFlat, layFlat2]` since they keep track of their children? Or do you need to specifically export every node in the tree?